gst/nnstreamer/tensor_converter/tensor_converter.c

```diff
@@ -250,8 +254,10 @@ gst_tensor_converter_class_init (GstTensorConverterClass * klass)
 
   /* set sink pad template */
   pad_caps = gst_caps_new_empty ();
+#ifndef NO_VIDEO
   /* video */
   gst_caps_append (pad_caps, gst_caps_from_string (VIDEO_CAPS_STR));
```

Comment:
In later versions, as suggested in today's chat, let's do
```
#ifdef NO_VIDEO
#define gst_caps_append (a, b) do { } while(0)
#else
#include <blahblah.h>
#endif
```
at the first lines of the code (or at converter header) to "centralize" #ifdef/#endif and don't use #ifdef/#endif in the middle of the code.

Reply:
add macros to set pad template and check media types